rosbag2_py/rosbag2_py/_message_definitions.pyi

```diff
@@ -0,0 +1,5 @@
+from typing import Any
+
+class LocalMessageDefinitionSource:
+    def __init__(self) -> None: ...
+    def get_full_text(self, arg0: str) -> Any: ...
```

Comment:
The linter doesn't like this.

Reply:
I wonder if the pyi files are generated automatically 🤔 

No matter: https://github.com/ros2/rosbag2/pull/1697/commits/0b6d099f71ca618d002279ddf20c7c79b9bac97f